src/app/dataset-flow/dataset-flow-details/tabs/flow-details-summary-tab/flow-details-summary-tab.component.ts

```diff
@@ -0,0 +1,35 @@
+import { ChangeDetectionStrategy, Component, Input } from "@angular/core";
+import { FlowOutcome, FlowSummaryDataFragment } from "src/app/api/kamu.graphql.interface";
+import { MaybeNullOrUndefined } from "src/app/common/app.types";
+import { DataHelpers } from "src/app/common/data.helpers";
+
+@Component({
+    selector: "app-flow-details-summary-tab",
+    templateUrl: "./flow-details-summary-tab.component.html",
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FlowDetailsSummaryTabComponent {
+    @Input() flowDetails: FlowSummaryDataFragment;
+    public readonly FlowOutcome: typeof FlowOutcome = FlowOutcome;
+
+    public outcomeClass(flowOutcome: MaybeNullOrUndefined<FlowOutcome>): string {
+        switch (flowOutcome) {
+            case FlowOutcome.Success:
+                return "success-outcome";
+            case FlowOutcome.Aborted:
+                return "aborted-outcome";
+            case FlowOutcome.Failed:
+                return "failed-outcome";
+            default:
+                return "";
```

Comment:
I think I've already saw a code like this in History or Main window.

Reply:
fixed